usaspending_api/api_docs/markdown/endpoints.md

```diff
@@ -173,6 +173,7 @@ The currently available endpoints are listed in the following table.
 |[/api/v2/search/spending_by_category/funding_subagency/](/api/v2/search/spending_by_category/funding_subagency/)|POST| Returns data that is grouped in preset units to support the Spending by Funding Subgency data visualizations on USAspending.gov's Advanced Search page |
 |[/api/v2/search/spending_by_category/naics/](/api/v2/search/spending_by_category/naics/)|POST| Returns data that is grouped in preset units to support the Spending by NAICS data visualizations on USAspending.gov's Advanced Search page |
 |[/api/v2/search/spending_by_category/psc/](/api/v2/search/spending_by_category/psc/)|POST| Returns data that is grouped in preset units to support the Spending by PSC data visualizations on USAspending.gov's Advanced Search page |
+|[/api/v2/search/spending_by_category/recipient/](/api/v2/search/spending_by_category/recipient/)|POST| Returns data that is grouped in preset units to support the Spending by Recipient DUNS data visualizations on USAspending.gov's Advanced Search page |
```

Comment:
Also either `/api/v2/search/spending_by_category/recipient/` needs to be updated to remove that trailing `/` or a trailing `/` needs to be added to the URL path. Currently the API docs link for this endpoint is broken.